File: week02/CHEFAPAR/main.go
package main

import "fmt"

func main() {
	var cases, n, temp, sum, last int
	for fmt.Scanf("%d", &cases); cases > 0; cases-- {
		sum = 0
		last = 1
		for fmt.Scanf("%d", &n); n > 0; n-- {
			fmt.Scanf("%d", &temp)
			if temp == 0 {
				sum += 1100
			} else if last == 0 {
				sum += 100
			}
			last = temp
		}
		fmt.Println(sum)
	}
}

File: week02/CHEFAPAR/main.c
#include <stdio.h>

int main(){
    int cases, n, temp, fail;
    long sum;
    for (scanf("%d",&cases); cases > 0; cases--) {
        sum = 0;
        fail = 0;
        for (scanf("%d",&n); n > 0; n--) {
            scanf("%d",&temp);
            if (temp == 1 && fail) {
                sum++;
            } else if (temp == 0) {
                sum += 11;
                fail = 1;
            }
        }
        printf( "%ld\n",(sum*100));
    }
    return 0;
}
